src/schema/firefox-schemas-import.js

```diff
@@ -32,7 +32,9 @@ const UNRECOGNIZED_PROPERTY_REFS = [
 ];
 
 const schemaRegexes = [
+  // eslint-disable-next-line prefer-regex-literals
   new RegExp('browser/components/extensions/schemas/.*\\.json'),
+  // eslint-disable-next-line prefer-regex-literals
   new RegExp('toolkit/components/extensions/schemas/.*\\.json'),
```

Comment:
Any reason for not changing this too? I guess too many '\' to escape? (anyway fine by me to disable it here)

Reply:
Yes, escaping backslashes in a path doesn't make the regexp very readable.